clinica/converters/aibl_to_bids/utils/json.py

```diff
@@ -0,0 +1,250 @@
+            [
+                "ModeOfAdministration",
+                ("RadiopharmaceuticalStartTime",),
+                "n/a",
+            ],  # uses "RadiopharmaceuticalStartTime" to decide on admin mode
+            # todo :  and (0054, 1002) Counts Source
+            ["InfusionRadioactivity", (), np.nan],  # todo : corresp ?
+            ["InfusionStart", ("RadiopharmaceuticalStartTime",), np.nan],
+            ["InfusionSpeed", (), np.nan],  # todo : corresp ?
+            ["InfusionSpeedUnits", (), "n/a"],  # todo : corresp ?
+            ["InjectedVolume", ("RadiopharmaceuticalVolume",), np.nan],
+            # Reconstruction
+            # todo : see(0054, 1000) Series Type ; (0054, 1001) Units ; (0028, 0051) Corrected Image
+            # see (0028, 1052) Rescale Intercept ; (0028, 1053) Rescale Slope
+            # (0018, 1100) Reconstruction Diameter ; (0018, 1149) Field of View Dimension(s) ; (0018, 1147) Field of View Shape
+            ["AcquisitionMode", (), "n/a"],
+            ["ImageDecayCorrected", ("DecayCorrection",), "n/a"],
+            ["ImageDecayCorrectionTime", ("DecayCorrection",), np.nan],
+            ["ReconMethodName", ("ReconstructionMethod",), "n/a"],
+            [
+                "ReconMethodParameterLabels",
+                ("ReconstructionMethod",),
+                [""],
+            ],  # todo : corresp more precise ?
+            ["ReconMethodParameterUnits", ("ReconstructionMethod",), [""]],
+            ["ReconMethodParameterValues", ("ReconstructionMethod",), [""]],
+            ["ReconFilterType", ("ConvolutionKernel",), "n/a"],
+            ["ReconFilterSize", ("ConvolutionKernel",), np.nan],  # todo : corresp ?
+            ["AttenuationCorrection", ("AttenuationCorrectionMethod",), "n/a"],
+        ],
+    ).set_index(keys="BIDSname", drop=False)
+
+
+def _get_dcm_value_from_header(
+    keys_list: Tuple[str], dicom_header: Union[FileDataset, DataElement]
+) -> Union[str, int, list, None]:
+    """
+    Get the value from the dicom header corresponding to a dicom Tag/key list
+
+    Parameters
+    ----------
+    keys_list :
+        Tuple representing where to get the expected information in the dicom header
+
+    dicom_header :
+        Dicom header to parse
+
+    Returns
+    -------
+        The value of the tag obtained from the header
+
+    """
+    # todo : how do i test this with sequences ?
+    if not keys_list:
+        # Handles case where is ()
+        return None
+
+    result = None
+    if len(keys_list) == 1:
+        result = dicom_header.get(Tag(keys_list[0]))
+    elif header := dicom_header.get(Tag(keys_list[0])):
+        # Handles nested tags
+        _get_dcm_value_from_header(keys_list[1:], header[0])
+    if result:
+        return result.value
+
+    # todo
+    # if result:
+    #     if type(result.value) == MultiValue:
+    #         # Handles case where result is MultiValue, which is not JSON serializable
+    #         return list(result.value)
```

Comment:
Maybe implement this in another function wrapping `_get_dcm_value_from_header` and handling the result value.